docs/user/document/distribution/tlp/label-usage.en.md

```diff
@@ -1,22 +1,22 @@
 # Label - Usage
 
-* The label should be set and must be valid. There are four color (or traffic light) values:
+The label should be set and be valid. There are four color (or traffic light) values:
 
-  * `RED` - *personal for named recipients only*:
+* `RED` - *personal for named recipients only*:
```

Comment:
Please stick closer to the TLP v1 definition: https://www.first.org/tlp/docs/tlp-v1.pdf

```suggestion
* `RED` - *Not for disclosure, restricted to participants only*:

Reply:
Updated the short descriptions as proposed. Should we also take the long descriptions from the linked resource?

Reply:
Not sure yet. 
@mfd2007, @sthagen: Any opinions whether we should use our own text or the official one from chapter 3 in https://www.first.org/tlp/docs/tlp-v1.pdf?

I guess we should somewhere at least reference the default definition; something like: "For more information about the TLP, refer to [the TLP website from FIRST](https://first.org/tlp/)."

Reply:
Added the last sentence. Will reference this discussion in a ticket.